maro/simulator/scenarios/data_center/common.py

```diff
@@ -67,25 +71,25 @@ class DecisionPayload:
     """Decision event in Data center scenario that contains information for agent to choose action.
 
     Args:
-        valid_pms (List[dict]): A list contains dictionaries with 'pm_id', 'cpu', 'mem'.
+        valid_pms (List[]): A list contains ValidPhysicalMachine object.
```

Comment:
if pm.info can be accessed through snapshot_list, pm.id is enough here

Reply:
I am a little bit confused. Isn't snapshot_list only updated based on the snapshot resolution? Then what if some ticks that the snapshots are not updated yet? How could the agent get the latest information through snapshot list?